packages/Algernon.package/ALGNamedWorldMenuEntry.class/instance/allEntries.st

```diff
@@ -0,0 +1,10 @@
+accessing
+allEntries
+	| entries |
+	
+	entries := OrderedCollection new.
+	
+	entries addAll: (self menuEntries inject: OrderedCollection new into: 
+		[ :accumulator :entry | accumulator addAll: {entry}, entry allEntries; yourself ]).
```

Comment:
Don't know if the code snippet here (or the one in ALGWorldMenuType) causes the following problem, but i think adding an extra `{entry}` causes the following situation:

When i search for a top-level World Menu Item (e.g. Extras or Apps) and tab into it (via the entries action), not only the entries of that top level are shown as sub-items, but i get an extra item for that `entries` action.

![image](https://user-images.githubusercontent.com/68427341/125428052-ef89c8c7-055d-4c7c-ac19-8c1197974beb.png)

As far as i can tell clicking on that item doesn't really do sth. Quite strange, is that intended?

Reply:
Yes, this is intended behavior to make it easier to search for entries whose name you know but are not sure in which menu they are.

Reply:
To my mind it's not intended, but fixing it would be too hard. It's not caused by this additional `{entry}` (that one is there for finding the top menu on the global search), since that's in `allEntries` but in the submenu view we only call `menuEntries`

Reply:
We thought about opening the menu morph on this action in order for it to at least do something, but found it unnecessary

Reply:
@bewee ahh okay. Yeah, i also quickly talked with Nils about that issue. We resolved the discussion as we also thought that `fixing would be too hard` points it out. Maybe that is stuff for our last sprint before the submission, but for know i think its okay that way 👍